commit 895a2ef1b23661d5fb73732c466de139e620a96a
Author: Thomas Gleixner <tglx@linutronix.de>
Date:   Sat Feb 25 10:27:37 2017 +0000

    mac80211_hwsim: Replace bogus hrtimer clockid
    
    commit 8fbcfeb8a9cc803464d6c166e7991913711c612c upstream.
    
    mac80211_hwsim initializes a hrtimer with clockid
    CLOCK_MONOTONIC_RAW. That's not supported.
    
    Use CLOCK_MONOTONIC instead.
    
    Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
    Signed-off-by: Johannes Berg <johannes.berg@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/wireless/mac80211_hwsim.c b/drivers/net/wireless/mac80211_hwsim.c
index d3bad57..0fd7d7e 100644
--- a/drivers/net/wireless/mac80211_hwsim.c
+++ b/drivers/net/wireless/mac80211_hwsim.c
@@ -2662,7 +2662,7 @@ static int mac80211_hwsim_new_radio(struct genl_info *info,
 
 	tasklet_hrtimer_init(&data->beacon_timer,
 			     mac80211_hwsim_beacon,
-			     CLOCK_MONOTONIC_RAW, HRTIMER_MODE_ABS);
+			     CLOCK_MONOTONIC, HRTIMER_MODE_ABS);
 
 	spin_lock_bh(&hwsim_radio_lock);
 	list_add_tail(&data->list, &hwsim_radios);
